styles/style.css

```diff
@@ -0,0 +1,754 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;700&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+}
+
+main {
+  background: #f7f7f9;
+  padding-top: 75px;
+}
+
+button:enabled {
+  background: none;
+  color: #396df2;
+}
+
+button:hover {
+  background: #5161f1;
+  color: white;
+}
+
+button:active {
+  background: #081dda;
+  color: white;
+}
+
+button:disabled {
+  background: #ebebff;
+  color: black;
+}
+
+header {
+  position: fixed;
+  width: 95vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 24px;
+  background-color: #fff;
+  box-shadow: 0 2px 6px 0 rgb(0 0 0 / 5%), 0 2px 15px 0 rgb(0 0 0 / 15%);
+}
```

Comment:
- [ ] Kindly fix the margin of the hamburger to match that of the figma template 

**Your design**
![gede 1](https://user-images.githubusercontent.com/85095702/158699395-47038f79-aef6-435e-8598-aef1c2e5947c.PNG)

**Figma template**
![figma](https://user-images.githubusercontent.com/85095702/158699490-d1866796-4875-4e66-abad-100091ff2166.PNG)

Reply:
@phelian23 This fixes the issues.